### ISSUE TITLE
Authentication Error?

### ISSUE THREAD

**Issue by richardrosales** [yzyio/adidas-multi-session#30]
I'm trying to test using http://tools.yzy.io/hmac.html as the PRODUCT_URL and keep getting stuck in a "Checking if session is past splash" loop. Seeing the following error before that though. Am I missing some sort of authentication step? 

```
Cache Access Denied.
Sorry, you are not currently allowed to request http://tools.yzy.io/ip.php from this cache until you have authenticated yourself.
Please contact the cache administrator if you have difficulties authenticating yourself.
```

**Comment by pfcpinto** [comment]
Is your IP authorized to access the proxies? are the proxies input in the correct manor USER:PASS@IP:PORT

**Comment by richardrosales** [comment]
@pfcpinto Thank you... that was it. I had a ':' instead of an '@' in the proxy list. Getting a browser popup now and the last message I'm seeing in terminal is:

`INFO: [CHROME/PROXY] Login with...`

Does that mean it's a success?

**Comment by pfcpinto** [comment]
yes @monarck did the browser pop up?

**Comment by pfcpinto** [comment]
if not make sure you have the chromedriver.exe and phantomjs.exe in the main folder

**Comment by richardrosales** [comment]
@pfcpinto Yeah the browser popped up to 2 new windows (tested with 2 proxies). They were pointing to the 'http://tools.yzy.io/hmac.html' test link but were white/blank screens. Wasn't sure if I'm supposed to be seeing any 'success' text or not. Both chromedriver and phantomjs are in the main folder.

Thanks for your help!

**Comment by pfcpinto** [comment]
you're squared away then

**Comment by ccarley** [comment]
I am receiving the same Authentication Error. But my proxies are formatted correctly with the username and password. Any idea to what i'm doing wrong?
